src/map.rs

```diff
@@ -4369,27 +4349,12 @@ mod test_map {
     fn test_raw_entry() {
         use super::RawEntryMut::{Occupied, Vacant};
 
-        let xs = [
-            (1i32, 10i32),
-            (2, 20),
-            (3, 30),
-            (4, 40),
-            (5, 50),
-            (6, 60),
-            (7, 70),
-            (8, 80),
-            (9, 90),
-            (10, 100),
-        ];
+        let xs = [(1i32, 10i32), (2, 20), (3, 30), (4, 40), (5, 50), (6, 60)];
```

Comment:
Why this change to have fewer elements? That seems fairly orthogonal to the remaining changes.

Reply:
Sorry, I just copied and pasted this test in hashbrown. Has been restored in the latest submission.